table/tables/cache.go

```diff
@@ -203,3 +232,20 @@ func (c *cachedTable) RemoveRecord(ctx sessionctx.Context, h kv.Handle, r []type
 	}
 	return c.TableCommon.RemoveRecord(ctx, h, r)
 }
+
+func (c *cachedTable) renewLease(ts uint64, op RenewLeaseType, data *cacheData) func() {
+	return func() {
+		tid := c.Meta().ID
+		lease := leaseFromTS(ts)
+		succ, err := c.handle.RenewLease(tid, lease, op)
+		if err != nil {
+			log.Warn("Renew read lease error")
+		}
+		if succ {
+			c.cacheData.Store(&cacheData{
```

Comment:
Start: oldStart

If you don't initialze it, it will be 0 and this is absolutely wrong